get_all_blobs/src/main.rs

```diff
@@ -0,0 +1,142 @@
+use std::{fs, str::FromStr};
+
+use alloy::{
+    dyn_abi::JsonAbiExt,
+    json_abi::JsonAbi,
+    network::Ethereum,
+    primitives::Address,
+    providers::{Provider, RootProvider},
+};
+use client::EigenClientRetriever;
+use serde::{Deserialize, Serialize};
+
+mod blob_info;
+mod client;
+mod generated;
+
+#[derive(Debug, Serialize, Deserialize)]
+struct BlobData {
+    pub commitment: String,
+    pub blob: String,
+}
+
+const EIGENDA_API_URL: &str = "https://disperser-holesky.eigenda.xyz:443";
+
+async fn get_blob(commitment: &str) -> anyhow::Result<Vec<u8>> {
+    let client = EigenClientRetriever::new(EIGENDA_API_URL).await?;
+    let data = client
+        .get_blob_data(&commitment)
+        .await?
+        .ok_or_else(|| anyhow::anyhow!("Blob not found"))?;
+    Ok(data)
+}
+
+async fn get_transactions(
+    provider: &RootProvider<
+        alloy::transports::http::Http<alloy::transports::http::Client>,
+        Ethereum,
+    >,
+    validator_timelock_address: Address,
+    commit_batches_selector: &str,
+) -> anyhow::Result<()> {
+    let latest_block = provider.get_block_number().await?;
+    let mut json_array = Vec::new();
+
+    for block_number in 0..=latest_block {
```

Comment:
For holesky testing we should not start at 0 or it will take too long, maybe this could be a parameter